### DISCUSSION TITLE
Admin Menu not shown

### DISCUSSION THREAD

**Discussion by matzeeg3** [requarks/wiki#5255] [Help / Questions]
Hi everybody,
i have a wiki.js installation running in Docker.
Everything looks fine for now.
I enabled AAD Login and if i login with the user i am able to login and view the page but:
The Icon on the top right corner keep saying "login" and not "Account" and i don´t see the "Admin" Link
If i than add /a/Pages or /a/Dashboard i can enter the Admin Area but i can not see the Menu.
How can i fix this?
Kind Regards
Matze

**Comment by NGPixel** [comment]
Try to login again?

**Comment by matzeeg3** [comment]
Done multiple times and with different browser. Same issue
